Project.toml

```diff
@@ -13,33 +13,30 @@ ImageIO = "82e4d734-157c-48bb-816b-45c225c6df19"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
 Proj4 = "9a7e659c-8ee8-5706-894e-f68f43bc57ea"
-# Proj = "c94c279d-25a6-4763-9509-64d165bea63e"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StructArrays = "09ab397b-f2b6-538f-b94a-2f83cf4a842a"
 
 [compat]
 Colors = "0.12"
+Downloads = "1.6"
 GeoInterface = "0.5"
 GeoJSON = "0.5"
 GeometryBasics = "0.4.1"
 ImageIO = "0.6"
 Makie = "0.17"
-Proj4="0.7"
-# Proj = "1.0"
+Proj4 = "0.7"
 Reexport = "1"
 StructArrays = "0.4, 0.5, 0.6"
 julia = "1.6"
 
 [extras]
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
-LightOSM = "d1922b25-af4e-4ba3-84af-fe9bea896051"
-OSMMakie = "76b6901f-8821-46bb-9129-841bc9cfe677"
+CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
 Downloads = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
 GLMakie = "e9467ef8-e4e7-5192-8a1a-b1aee30e663a"
-# Graphs = "86223c79-3864-5bf0-83f7-82e725a168b6"
-# GraphMakie = "1ecd5474-83a3-4783-bb4f-06765db800d2"
-CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
+LightOSM = "d1922b25-af4e-4ba3-84af-fe9bea896051"
+OSMMakie = "76b6901f-8821-46bb-9129-841bc9cfe677"
```

Comment:
@asinghvi17 I think its not a good idea to leave commented code in the dependencies :D